doc/source/building_images/build_expandable_disk.rst

```diff
@@ -389,3 +389,30 @@ oemconfig.oem-unattended Element
   automatically without requiring user interaction. If multiple
   possible target devices are discovered the image is deployed to
   the first device. ``kiwi_oemunattended`` in the initrd
+
+.. _installmedia_customize:
+
+Installation Media Customization
+--------------------------------
+
+The installation media created for OEM network or CD/DVD deployments can
+be customized with the `installmedia` which is a child section of the `type`
```

Comment:
...customized with the installmedia section which is ...

I think a word is missing